devHelper/README.md

```diff
@@ -1,7 +1,7 @@
 # How to generate mock data and start developing in your local 
 
 ## Step.1 start elasticsearch
-
+In this directory `(/devHelper)`, do:
 ```
 docker-compose -f ./esearch.yml up -d
```

Comment:
I think this should be

```suggestion
docker-compose -f ./docker/esearch.yml up -d
```

Reply:
yeah you are right!

actually i'll rewrite that part a bit. there is no need to jump between these directories like a 🐰

Reply:
Haha perf